mux/state.go

```diff
@@ -110,12 +109,10 @@ func (b *StateBroadcaster) Close() {
 }
 
 // Submit a new object to all subscribers
-func (b *StateBroadcaster) Submit(k interface{}, v interface{}) error {
+func (b *StateBroadcaster) Submit(k interface{}, v interface{}) {
 	if b != nil && k != nil {
```

Comment:
should we panic here, so users know they're not using the library correctly?

Reply:
yes